src/scripts.js

```diff
@@ -2,41 +2,21 @@ import './css/base.scss';
 import './css/styles.scss';
 
 import userData from './data/users';
-import activityData from './data/activity';
 import sleepData from './data/sleep';
 import hydrationData from './data/hydration';
-
+import HydrationRepo from './hydrationRepo'
+import SleepRepo from './SleepRepo.js'
 import UserRepository from './UserRepository';
 import User from './User';
 import Activity from './Activity';
 import Hydration from './Hydration';
 import Sleep from './Sleep';
 
+// import activityData from './data/activity';
+import fetchCalls from './apiCalls';
+// console.log(fetchCalls)
 
-
-let userRepository = new UserRepository();
```

Comment:
I believe this will be our only global variable aside from our dom elements once we are done refactoring.